SwiftUI-Template/SwiftUI-Template/Views/ActivityView.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  ActivityView.swift
+//  SwiftUI-Template
+//
+//  Created by Vitaliy Iakushev on 02.05.2024.
+//
+
+import SwiftUI
+
+struct ActivityView: UIViewControllerRepresentable {
+    
+    typealias UIViewControllerType = UIActivityViewController
+    
+    public var activityItems: [Any]
+    public var applicationActivities: [UIActivity]?
+    
+    func makeUIViewController(context: Context) -> UIActivityViewController {
+        let vc = UIActivityViewController(activityItems: activityItems, applicationActivities: applicationActivities)
```

Comment:
```suggestion
        let viewController = UIActivityViewController(activityItems: activityItems, applicationActivities: applicationActivities)
```
Женя говорил не называть так переменные vc